apps/institution-service/openApi.json

```diff
@@ -379,6 +374,30 @@
                           }
                         }
                       ]
+                    },
```

Comment:
Regarding the openapi spec, there are a few errors, according to the swagger editor. 
<img width="684" alt="image" src="https://github.com/user-attachments/assets/0645d0a8-d980-4347-a9a1-32150e29d28a">

Reply:
I'm not sure if those errors are all that important to fix, but wanted to just make a note.

Reply:
Fixed